BD-3004: Add recommendation for incremental updates (#7981)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/cloud_ingestion/overview.md

You can set it to `null` if you want to omit an attribute from a user's profile. If you want an attribute to remain unchanged, don't send it to Braze until it's been updated. To completely remove an attribute, use `TO_JSON(OBJECT_CONSTRUCT_KEEP_NULL(...))`.

### Make incremental updates

> Make incremental updates to make sure your data is updated to prevent any overwrites. When simultaneous updates are made, there may be unintentional overwrites in the data.

Reviewer: ```suggestion
Make incremental updates to your data so you can prevent unintentional overwrites when simultaneous updates are made.
```

Reviewer: Combining both sentences together, but feel free to tweak my suggestion as you see fit.

> Consider the following example where a user has two attributes:

Reviewer: ```suggestion
In the following example, a user has two attributes:
```

- Color: "Green"
- Size: "Large"

> Now, let's say we receive the following two simultaneous requests, and both requests are initiated at the same time.

Reviewer: ```suggestion
Then Braze receives the following two updates to that user simultaneously:
```

- Request 1: Change color to "Red"
- Request 2: Change size to "Medium"

> In this example, Request 1 occurs first, so the attributes are updated to:

Reviewer: ```suggestion
Because Request 1 occurs first, the user's attributes are updated to the following:
```

- Color: "Red"
- Size: "Large"

> However, when Request 2 occurs, we begin with the original attribute values ("Green" and "Large"). This updates the attributes to the following:

Reviewer: ```suggestion
However, when Request 2 occurs, Braze starts with the original attribute values ("Green" and "Large"), then updates the user's attributes to the following:
```

- Color: "Green"
- Size: "Medium"

> When these requests have finished, Request 2 has overwritten the update from Request 1, so it's best practice to stagger updates to prevent requests to be overwritten.

Reviewer: ```suggestion
When the requests are finished, Request 2 will overwrite the update from Request 1. Due to this, it's best to stagger your updates so you can prevent requests from being overwritten.
```
